app/templates/components/session-overview.hbs

```diff
@@ -28,102 +28,70 @@
-      <br />
-      <div class="inline-label-data-block sessiondescription">
-        <label>{{t "general.description"}}:</label>
-        <div>{{inplace-textarea value=session.sessionDescription.description save='changeDescription'}}</div>
-      </div>
-    {{else}}
-      <div class="inline-label-data-block">
-        <label>{{t "sessions.type"}}:</label>
-        <div>{{session.sessionType.title}}</div>
-      </div>
-      <br />
-      <div class="inline-label-data-block">
-        <label>{{t "sessions.supplementalCurriculum"}}:</label>
-        <div>{{boolean-check value=session.supplemental}}</div>
-      </div>
-      <div class="inline-label-data-block">
-        <label>{{t "sessions.specialAttireRequired"}}:</label>
-        <div>{{boolean-check value=session.attireRequired}}</div>
+    <div class="form-col-12 sessiontype">
+      <label class="form-label">{{t "sessions.type"}}:</label>
+      <div class="form-data">
+        {{inplace-select
+          value=session.sessionType.id
+          options=sortedSessionTypes
+          save='changeSessionType'
+        }}
       </div>
-      <div class="inline-label-data-block">
-        <label>{{t "sessions.specialEquipmentRequired"}}:</label>
-        <div>{{boolean-check value=session.equipmentRequired}}</div>
-      </div>
-      <br />
-      <div class="inline-label-data-block">
-        <label>{{t "sessions.independentLearning"}}:</label>
-        <div>{{boolean-check value=isIndependentLearning}}</div>
-      </div>
-      <br />
-      <div class="inline-label-data-block">
-        <label>{{t "general.description"}}:</label>
-        <div>{{session.sessionDescription.description}}</div>
+    </div>
+    <div class="form-col-12 sessionsupplemental">
+      <label class="form-label">{{t "sessions.supplementalCurriculum"}}:</label>
+      <div class="form-data">{{inplace-boolean value=session.supplemental save='changeSupplemental'}}</div>
+    </div>
+    <div class="form-col-6-left sessionspecialattire">
+      <label class="form-label">{{t "sessions.specialAttireRequired"}}:</label>
+      <div class="form-data">{{inplace-boolean value=session.attireRequired save='changeSpecialAttire'}}</div>
+    </div>
+    <div class="form-col-6-right sessionsepcialequiptment">
+      <label class="form-label">{{t "sessions.specialEquipmentRequired"}}:</label>
+      <div class="form-data">{{inplace-boolean value=session.equipmentRequired save='changeSpecialEquipment'}}</div>
+    </div>
+    <div class="form-col-4-left">
+      <label class="form-label">{{t "sessions.independentLearning"}}:</label>
+      {{!-- <div class='form-data independentlearningcontrol' {{action 'toggleIndependentLearning'}}>{{inplace-boolean value=session.isIndependentLearning}}</div> --}}
+      <div class="form-data" {{action 'toggleIndependentLearning'}} class='control form-data switch yes-no switch-green'>
+        <input {{ bind-attr checked=session.isIndependentLearning }} type='checkbox' class="switch-input" />
```

Comment:
I think this should probably be the component inplace-boolean now that it is working to avoide duplication, it will require rewriting actions.toggleIndependantLearning in session-overview.js to take a value instead of toggling back and forth.